modules/rosa-hcp-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,272 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-hcp-aws-rhcos-ami_{context}"]
+= {hcp-title} AMIs
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {hcp-title} nodes.des.
+
+[id="rosa-hcp-aws-rhcos-ami-415_{context}"]
+== Version 4.15 AMIs
```

Comment:
Comparing the output of 'rosa list versions --hosted-cp -o json|jq -r '[.[]|select(.raw_id | contains("4.15"))]'|jq -r '.[0].image_overrides.aws[]| "(.region.id):(.ami)"''. This document PR misses bellow region ami:
ap-northeast-3:ami-001bac2f632acb80b
ca-west-1:ami-01cd726ce7cf92d4e
us-gov-east-1:ami-0c0f71174a3fda3fe
us-gov-west-1:ami-075352eb70816ce65

And ap-north-3:ami-02e77a92d0699aec0 don't exist in the backend output.

Others ami ids on their regions are different.

Reply:
@michaelryanmcneill, @iamkirkbater, and @yuwang-RH same clarifying questions as the 4.14 section